project/AppDependencies.scala

```diff
@@ -5,18 +5,18 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "3.4.0",
+    "uk.gov.hmrc"             %% "bootstrap-backend-play-27"  % "5.3.0",
     "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-27"         % "0.50.0"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "3.4.0"                % Test,
-    "org.scalatest"           %% "scalatest"                % "3.0.9"                 % Test,
+    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "5.3.0"                % Test,
+    "org.scalatest"           %% "scalatest"                % "3.2.9"                 % Test,
     "com.typesafe.play"       %% "play-test"                % current                 % Test,
     "org.pegdown"             %  "pegdown"                  % "1.6.0"                 % "test, it",
-    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.35.10"               % "test, it",
+    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.36.8"               % "test, it",
```

Comment:
In the FE found removing flexmark didnt break anything, do you know if it's needed?

Reply:
seems to break the backends, so will keep it